Tests/test_NCBIXML.py

```diff
@@ -1746,7 +1747,7 @@ def test_xml_2222_blastp_001(self):
         handle.close()
 
     def test_xml_2218L_rpsblast_001(self):
-        """Parse PSI-BLASTP 2.2.18, single query which converges in 3 iterations (xml_2218L_rpsblast_001)."""
+        """Parse RSP-BLAST 2.2.18, single query which converges in 3 iterations (xml_2218L_rpsblast_001)."""
```

Comment:
Can you delete the comment after this too (the bit about pgpblast vs psiblast, which I assume was copy-and-pasted in error from an older test).